services/121-service/src/shared/types/ui-language-translation-partial.type.ts

```diff
@@ -0,0 +1,14 @@
+import { UILanguage } from '@121-service/src/shared/enum/ui-language.enum';
+import { TranslationPartial } from '@121-service/src/shared/types/translation-partial.type';
+
+/**
+ * An object that contains 0..n string translations for languages the UI
+ * supports. Use this if you want to allow incomplete translations.
```

Comment:
can this (and the preferred lanuage variant of this) use a bit more explanation? Maybe some example of full vs partial? In fact, the 'full' variant is only used for preferred languages and only for the linguonyms right? Doesn't this put a bit of unjustified focus on the partial nature here, with also the phrase 'allowing incomplete translations' sound a big negative, while it's in fact the default apparently.